ABOUT.md

```diff
@@ -0,0 +1,51 @@
+
+# What is MineColonies?
+Here, we would like to give you an overview of what our mod is.
+
+### Creativity 
+MineColonies is a town building mod that allows you create your own thriving Colony within Minecraft. It tends to depict a real-life scenario by providing you with different tools in your chest to aid your gaming experience. Featuring many NPC workers such as Builders, Crafters, Farmers, Guards, Enchanters and Alchemists, MineColonies provides the ability to create a rich and unique Colony.
+
+### Governance
+In MineColonies, you take control of what goes on in your Colony. Have an adequate number of villagers who will live in the town with you. Create a division of labour, hire workers to help in developing the Colony while you maturely act as a Supervisor.
```

Comment:
I would add/write: "You have complete control of how you build your Town. You determine how many workers you want or need, what type of workers, where they will live, hire or fire as you see fit. Creating a Town at your own pace, your own way. It is your Town to design and develop, you are the designer, administrator and boss always."